src/domains/citizen-auth-app/09_function_profile.tf

```diff
@@ -415,12 +415,12 @@ resource "azurerm_monitor_autoscale_setting" "function_profile" {
   }
 
   profile {
-    name = "wallet_gate0"
+    name = "auth_${module.common_values.scaling_gate.name}"
```

Comment:
```suggestion
    name = "wallet_${module.common_values.scaling_gate.name}"
```

Reply:
I actually set `auth_` to give it a generic value, so we wouldn’t have to make further code changes, leaving the configuration in `common_values` module. If we want to specify `wallet_`, I would suggest removing `auth_` altogether and directly changing the name in `common_values` to `wallet_gate1` instead of `gate1`. What do you think?